src/Raven.Server/Documents/TransactionOperationsMerger.cs

```diff
@@ -794,7 +1031,10 @@ private void RunEachOperationIndependently(List<MergedTransactionCommand> pendin
                         }
                         catch (Exception e)
                         {
-                            if(alreadyRetried == false && op.RetryOnError)
+                            //Todo I'm not sure for what is for...
+                            _recordingStatus?.Record(TxInstruction.Rollback);
```

Comment:
We need to know that this happened, but in general a dispose without commit will also work, so we can skip it

Reply:
done